### PR TITLE
Kirill.sitcevoy/ef code first

### PR DESCRIPTION
**Pull request by kievzenit**
Entity Framework home task using code first approach

### CODE REVIEW

File: EFCodeFirst/EFCodeFirst/Migrations/Configuration.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    using System;
+    using System.Data.Entity;
+    using System.Data.Entity.Migrations;
+    using System.Linq;
+
+    internal sealed class Configuration : DbMigrationsConfiguration<EFCodeFirst.DB.Context>
+    {
+        public Configuration()
+        {
+            AutomaticMigrationsEnabled = false;
```
Comment (Artya) [inline EFCodeFirst/EFCodeFirst/Migrations/Configuration.cs:12]: do this one was autogenerated?

File: EFCodeFirst/EFCodeFirst/Program.cs [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+            };
+
+            context.Suppliers.Add(supplier1);
+            context.Suppliers.Add(supplier2);
+
+            context.SaveChanges();
+        }
+        static void prepareProducts(Context context)
+        {
+            var suppliers = context.Suppliers.Take(2).ToArray();
```
Comment (Artya) [inline EFCodeFirst/EFCodeFirst/Program.cs]: you can create suppliers here as well and just assign to the product field, then EF will save suppliers and products as well by one request without this trick

File: EFCodeFirst/EFCodeFirst/Program.cs [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+
+            context.Products.Add(product1);
+            context.Products.Add(product2);
+            context.Products.Add(product3);
+
+            context.SaveChanges();
+        }
+        static void prepareOrders(Context context)
+        {
+            var products = context.Products.Take(2).ToArray();
```
Comment (Artya) [inline EFCodeFirst/EFCodeFirst/Program.cs]: same here

File: EFCodeFirst/EFCodeFirst/Program.cs (hunk 2 above)
Comment (kievzenit) [reply EFCodeFirst/EFCodeFirst/Program.cs]: Done in cd93496ab08d07c56a756b70c234cd198533c6a4

File: EFCodeFirst/EFCodeFirst/Program.cs (hunk 3 above)
Comment (kievzenit) [reply EFCodeFirst/EFCodeFirst/Program.cs]: Done in cd93496ab08d07c56a756b70c234cd198533c6a4

File: EFCodeFirst/EFCodeFirst/Migrations/Configuration.cs (hunk 1 above)
Comment (kievzenit) [reply EFCodeFirst/EFCodeFirst/Migrations/Configuration.cs:12]: Honestly, I don't remember

**Pull request closed without merging**